tests/remote/main.tftest.hcl

```diff
@@ -1,12 +1,12 @@
-provider "azurerm" {
-  features {}
-}
```

Comment:
Removing this provider block will break if you have tests without a `module {}` block. We should define the provider in the test file, not in `./tests/remote/providers.tf'.